esque/cli/commands.py

```diff
@@ -1118,3 +1120,134 @@ def ping(state: State, times: int, wait: int):
     click.echo("--- statistics ---")
     click.echo(f"{len(deltas)} messages sent/received.")
     click.echo(f"min/avg/max = {min(deltas):.2f}/{(sum(deltas) / len(deltas)):.2f}/{max(deltas):.2f} ms")
+
+
+@esque.command("io")
+@click.option("-i", "--input-uri", help="Input URI", metavar="<input_uri>", required=True)
+@click.option(
+    "-o",
+    "--output-uri",
+    help="Output URI",
+    metavar="<output_uri>",
+    default="pipe+json://stdout?kv__indent=2&h__skip_marker=1",
+)
+def io(input_uri: str, output_uri: str):
+    """Run a message pipeline.
+
+    Read all messages from the input configured by <input_uri> and write them to the output configured by <output_uri>.
+
+    \b
+    URI Format:
+    <handler scheme>+<key serializer scheme>[+<value serializer scheme>]://<host>/<path>[?<param 1>(&<param 2>)]
+
+    \b
+    <handler scheme>    - Scheme that defines which handler should be used.
+                          (See "Available Handlers" below)
+    <serializer scheme> - Define data serialization format for key and value in message (e.g json or avro).\b
+                          If no value serializer scheme defined key scheme will be used for both.
+                          (See "Available Serializers" below)
+    <host> and <path>   - Depends on handler and defines the exact source or target of the data pipeline.
+                          (See "Available Handlers" below)
+    <param>             - Define parameters for handler and/or serializer. Use the following prefixes to define
+                          which component a parameter should be used for:
+                            k__  use as key serializer parameter
+                            v__  use as value serializer parameter
+                            kv__ use as key and value serializer parameter
```

Comment:
It is not obvious to me when a parameter would have the `kv__` prefix

Reply:
Is it more clear now? :slightly_smiling_face:

Reply:
Yeah! :)